sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_impl.hpp

```diff
@@ -106,10 +140,15 @@ double joint_matmul(TOperand *A, TOperand *B, TResult *C, queue &q, int i) {
 #else
                 ; // no initialization
 #endif
-
+#ifdef VNNI
```

Comment:
Since default meaning is changed (compilation without VNNI macro is now using row_major B instead of packed), please, update all other .cpp files, which are using this .hpp file to keep testing coverage as before.